readthedocs/oauth/clients.py

```diff
@@ -0,0 +1,71 @@
+from datetime import datetime
+
+import structlog
+from django.utils import timezone
+from requests_oauthlib import OAuth2Session
+
+log = structlog.get_logger(__name__)
+
+
+def _get_token_updater(token):
+    """
+    Update token given data from OAuth response.
+
+    Expect the following response into the closure::
```

Comment:
Not sure what this is saying -- do you mean that is the expected value of the `token` arg?

Reply:
Nope, this is likely the response you get when the request for a new token is made.